web/js/doppelkopf.js

```diff
@@ -5,19 +5,42 @@ function addPlayers(select, players) {
     option.appendChild(document.createTextNode(player.name));
     select.appendChild(option);
   }
+  for (let i = 1; i < 5; i += 1) { // Vorauswahl auf empty setzen
+    document.getElementById(`player${i}`).selectedIndex = -1;
```

Comment:
Template-String gefällt mir gut!

Reply:
Diese Schleife ist gefühlt unnötig - du gehst ja in Zeile 18 schon durch alle Spieler durch, und hier dann nochmal durch alle, damit hast du nicht mehr nur 4 Schleifendurchläufe sondern 16. Außerdem hast du das Element ja schon (du machst unten bereit `document.getElementById`), also ist das eigentlich auch doppelt.

Du kannst entweder hier einfach `select.selectedIndex = -1` machen, dann meckert womöglich der Linter, oder du machst unten sowas wie

```
const players = JSON.parse(request.responseText);
const playerSelect = document.getElementById(`player${i}`;
addPlayers(playerSelect, players);
playerSelect.selectedIndex = -1;
```